pavelib/assets.py

```diff
@@ -906,7 +856,6 @@ def watch_assets(options):
     sass_directories = get_watcher_dirs(theme_dirs, themes)
     observer = Observer(timeout=wait)
 
-    CoffeeScriptWatcher().register(observer)
```

Comment:
**and** we get to kill the coffeescript watcher? This PR keeps getting better and better